client/package.json

```diff
@@ -38,7 +38,7 @@
     "karma-ng-html2js-preprocessor": "^0.1.0",
     "lodash": "^2.4.1",
     "karma-coverage": "^0.2.0",
-    "protractor": "~1.6.1",
+    "protractor": "actionless/protractor#old-chrom",
```

Comment:
it's the latest protractor but the thing what chromedriver version is hardcoded and in order to use travis we need older version of chromedriver (travis have chromium 37 installed while newer chromedriver requires 38)

and i docker-based travis workers we can't do operations which require super user rights so it's not possible to install newer chrome